packages/web-components/fast-foundation/src/utilities/style/icons.ts

```diff
@@ -0,0 +1,47 @@
+export const chevronUpIcon = `<svg width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think this file should be moved somewhere else. It's not really a "utility".

Reply:
Perhaps pull "style" up out of the "utilities" folder.

Reply:
I moved the icons to the root and don't export them by default. I could go either way if they are "styles" or not. I think the current "styles" constitutes at least some form of a "utility" though.